Services/Mail/classes/Message/class.ilMailBody.php

```diff
@@ -9,7 +9,7 @@ class ilMailBody
     /**
      * @var mixed|null|string|string[]
      */
-    private $bodyContent;
+    private mixed $bodyContent;
```

Comment:
The `mixed` type is not supported in PHP 7.4, which we'll have to support as well in ILIAS 8.